meta/collection-requirements.yml

```diff
@@ -1,3 +1,5 @@
 # SPDX-License-Identifier: MIT
 collections:
-  - containers.podman
+  - name: https://github.com/containers/ansible-podman-collections.git
+    type: git
+    version: master
```

Comment:
Note - this is a hack until a new version of containers.podman is published with support for podman 4.1.x